src/lib/controllers/configuration-manager/generators/generate-dh-param.d.ts

```diff
@@ -0,0 +1 @@
+export function generateDHParam(bits: number): string;
```

Comment:
Why not just convert `generate-dh-param.js` to TypeScript?

Reply:
No proper types for this npm module. We can rewrite it in typescript at some point, but the scope of this PR was to have something asap for moving to k8s.